src/stories/PageBanner.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { checkA11y } from '@storybook/addon-a11y';
+import { IntlProvider } from 'react-intl';
+
+
+import PageBanner from 'js/modules/PageBanner';
+
+import dummyImage from 'stories/images/pagebanner_test.jpg';
+
+storiesOf('PageBanner', module)
+  .addDecorator(checkA11y)
```

Comment:
Since we're adding checkA11y and IntlProvider on all components, I think it makes sense to do this globally once. 
You can also add a decorator globally for all stories like this in .storybook/config.js according to the docs
```
import React from 'react';
import { configure, addDecorator } from '@storybook/react';
import 'css/coa.css'

const SomeDecorator = (storyFn) => (
  <div style={{
    border: '10px solid burlywood'
  }}>
    { storyFn() }
  </div>
);

addDecorator(SomeDecorator);

// automatically import all files ending in *.stories.js
const req = require.context('../src/stories', true, /.stories.js$/);
function loadStories() {
  req.keys().forEach((filename) => req(filename));
}

configure(loadStories, module);

```

Reply:
👑